operations/tasks/upgrade-docker.yml

```diff
@@ -0,0 +1,49 @@
+---
+- name: Mark node as unschedulable
+  command: >
+    oadm manage-node {{ inventory_hostname }} --schedulable=false
+
+- name: Wait until no more pods are running in node
+  register: active_pods_result
+  shell: >
+    oadm manage-node {{ inventory_hostname }} --list-pods |
```

Comment:
It would be easier to filter by 'READY' column - if it doesn't start with '0/' there are running pods:
`oadm manage-node {{ inventory_hostname }} --list-pods --no-headers | grep -v 'READY' | awk '{print $2}' | grep -v '0/'`

Reply:
Nice! 

Note: `--no-headers` doesn't seem to work for `oadm` :( But you probably know that thus the `grep -v 'READY'`.

Reply:
Yes, this is weird, maybe a bug in oc